TypeScript.py

```diff
@@ -281,6 +281,9 @@ def __init__(self):
         self.fileMap = {}
         self.refInfo = None
         self.versionST2 = False
+        self.tempFileMap = {}
+        self.tempFileList = []
```

Comment:
The mapping is used in the receiver function, so that upon completion of the reload, the temporary file name corresponding to the request sequence number can be re-used.